boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -153,6 +153,17 @@
 	status = "okay";
 };
 
+/* StemmaQT/Qwiic/Groove external connector */
+&i2c0 {
```

Comment:
These connectors are becoming quite popular, maybe we should start having a well-known alias for it so one could do a shield that refers to it, what do you think @erwango?